assets/javascripts/modules/react/directDebit.jsx

```diff
@@ -0,0 +1,146 @@
+                </p>}
+            </dd>
+        </div>
+    }
+}
+
+class AccountNumber extends React.Component {
+    constructor(props) {
+        super(props)
+
+    }
+
+
+    render() {
+        let invalid = this.props.valid === false;
+        return <div>
+            <dt className="mma-section__list--title">
+                <label className="label" for="payment-account">Account number</label>
+            </dt>
+            <dd className="mma-section__list--content">
+                <input value={this.props.value}
+                       onChange={this.props.onChange}
+                       id="payment-account"
+                       name="payment.account"
+                       pattern="[0-9]*" minlength="6"
+                       maxlength="10"
+                       className="input-text"
+                />
+                {invalid && <p className="mma-error">
+                    Please enter a valid bank account number
+                </p>}
+            </dd>
+        </div>
+    }
+}
+
+class AccountHolder extends React.Component {
+    constructor(props) {
+        super(props)
+
+    }
+
+    /*
+     * BACS requirement:
+     "The payer’s account name (maximum of 18 characters).
+     This must be the name of the person who is paying the Direct Debit
+     and has signed the Direct Debit Instruction (DDI)."
+     http://www.bacs.co.uk/Bacs/Businesses/Resources/Pages/Glossary.aspx
+     * */
+
+    render() {
+        let invalid = this.props.valid === false;
+
+        return <div>
+            <dt className="mma-section__list--title">
+                <label for="payment-holder">Account holder</label>
+            </dt>
+            <dd className="mma-section__list--content">
+                <input
+                    id="payment-holder"
+                    value={this.props.value}
+                    onChange={this.props.onChange}
+                    className="input-text"
+                />
+                {invalid && <p className="mma-error">
+                    This field is required
+                </p>}
+            </dd>
+        </div>
+    }
+}
+
+class Confirmation extends React.Component {
+    constructor(props) {
+        super(props)
+
+    }
+
+    render() {
+        let invalid = this.props.valid === false;
+        return <div>
+            <dt className="mma-section__list--title">
+                Confirmation
+            </dt>
+            <dd className="mma-section__list--content">
+                <label className="option">
+                <span className="option__input">
+                    <input type="checkbox"
+                           onChange={this.props.onChange}
+                           checked={this.props.checked}
+                    />
+                </span>
+                    <span className="option__label">
+                    I confirm that I am the account holder and I am solely able to authorise debit from the account
```

Comment:
There might be other text which needs to be brought over too. The renewal flow will need to be signed off by the legal team.